src/createUseBinding.lua

```diff
@@ -1,5 +1,5 @@
 local function createUseBinding(roact, useValue)
-	return function(defaultValue)
+	return function<T>(defaultValue: T): (any, (newValue: T) -> ())
```

Comment:
Could we use `typeof(Roact.createBinding(defaultValue))` here? In case Roact gets types?

Reply:
This file has no actual knowledge of Roact, there is just a variable named `roact`, which Luau will never be able to infer as the type of Roact, as there is nothing in particular connecting them together. Hence `roact` will always be `any`, and then `typeof(Roact.createBinding())` will also always be any.

This *could* work if we actually imported Roact somewhere, and then used the import

Reply:
You're right.